src/app/loadout-builder/LockArmorAndPerks.tsx

```diff
@@ -162,16 +162,14 @@ function LockArmorAndPerks({
   const storeIds = stores.filter((s) => !s.isVault).map((s) => s.id);
   const bucketTypes = buckets.byCategory.Armor.map((b) => b.type!);
 
-  const anyLocked = Object.values(lockedMap).some((lockedItems) =>
-    Boolean(lockedItems && lockedItems.length > 0)
-  );
+  const anyLocked = Object.values(lockedMap).some((lockedItems) => Boolean(lockedItems?.length));
 
   return (
     <div>
       <div className={styles.area}>
-        {((flatLockedMap.perk && flatLockedMap.perk.length > 0) ||
-          (flatLockedMap.mod && flatLockedMap.mod.length > 0) ||
-          (flatLockedMap.burn && flatLockedMap.burn.length > 0)) && (
+        {(flatLockedMap.perk?.length ||
+          flatLockedMap.mod?.length ||
+          flatLockedMap.burn?.length) && (
```

Comment:
These need to stay `> 0` because even though 0 is falsy, React will render it out as a 0.

Reply:
Should we write these ones that are for React as `>= 1` instead

Reply:
that is a really cool idea

Reply:
new idea: length implicitly can't be negative,
so actually `?.length !== 0`

• describes the idea of "needs some contents"
• allows the nice chaining notation
• meets the lint rule preventing things like `number | undefined > 0`